pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
No trip format is needed for writing pO2, sf (unless svv, but this is really not needed now), OER etc. 

So, a simple pickle read/write method could be added to master class, if this really should be a part of the class.

```
c = Cube(...)
c.save()
c.load()
```